test/ComponentTests/PlatformServices.Desktop.Component.Tests/DesktopTestSourceHostTests.cs

```diff
@@ -105,7 +96,7 @@ private static string GetTestAssemblyPath(string assemblyName)
         return Path.Combine(testAssetPath, assemblyName);
     }
 
-    private Mock<IRunSettings> GetMockedIRunSettings(string runSettingxml)
+    private static Mock<IRunSettings> GetMockedIRunSettings(string runSettingxml)
```

Comment:
please can you explain, why we need it to be static?

Reply:
Just took the liberty to address some code analyzers recommendation (method doesn't use any member so can be made static. `static` members are supposed to be faster so it's good to use them when possible).